paper.tex

```diff
@@ -2056,6 +2003,95 @@ \subsubsection{Future Work}
 \clearpage
 
 \appendices
+
+% TODO should this really go in the appendix?
+\section{Version Control}
+\label{sec:versionControl}
+
+Over the course of the project, we had to develop multiple applications in
+different languages. In total, we opened five different repositories: the blog,
+the \LaTeX source code for our paper, the Arduino code for the heart project,
+and of course one for Editour and another for ARuko. These five repositories
+span across multiple different markup and programming languages, including C\#,
+C++, JavaScript, HTML, CSS (and the \LaTeX markup language used to typeset this
+paper.) This gave us a welcome amount of practice using git for version control,
+as well as GitHub's other features oriented around coordinating a team.
+
+All three members of the team were contributing code and updating assets such as
+graphics and sound. Code complexity grew quickly as more features were added
+across the entire software stack. GitHub and git enabled us to view the history
+of the code and revert back to working versions when absolutely necessary. We
+researched the proper methodology of using version control in a small team,
+applying this methodology as best we could. This enabled us to work together
+efficiently, track down bugs, undo mistakes, and sometimes review each other's
+code for quality when time allowed.
+
+\subsection{GitHub Tools}
+\label{sec:gitHubTools}
+
+Some members of our team were very excited by the workflows GitHub's tools
+enabled. In this section, we will discuss some of the tools, some of which
+greatly aided in our organization and productivity.
+
+Initially, we experimented with using a four-column Kanban board to keep track
+of our progress. Figure~\ref{fig:kanbanBoard} shows a screenshot of this page on
+GitHub.
+
+The Kanban technique stems from Toyota in the 1940s. Kanban means ``visual
+signal'' or ``card'' in Japanese, which describes the core component of the
+technique~\autocite{terry2019}. At Toyota, line-workers would move colored cards
+to notify other teams that parts needed for assembly work were in short supply.
+Now, Kanban boards are integral to Agile software development. Still, some teams
+use physical Kanban boards, often using sticky notes and whiteboards.
+
+The advantage of using a Kanban board is that each feature or bug that
+is being worked on can be assigned directly to a collaborator on the project. It
+is then that person's responsibility to work on those tasks and update the
+status of those tasks on the board. We abandoned this when moving onto the main
+project because we came to realize that this structure was not necessary---there
+were only three members on the team, and we were almost always in the same room
+when working on the project. For Editour, we tracked features and issues using
+the Issues page instead, which is discussed in
```

Comment:
Should "Issues" be capitalized?

Reply:
I put it in quotes because it's the name of the tab. Maybe this isn't the correct way to do it though. (I do that elsewhere.)